recast_helloworld/steps.yml

```diff
@@ -13,7 +13,5 @@ rule hello_world:
         shell(
             "echo Hello my Name is {params.name} | tee {params.outputfile}"
         )
-    container_img:
-        "busybox"
-    container_:
+    container:
```

Comment:
REANA doesn't work with `busybox` (c.f. https://gitlab.cern.ch/recast-atlas/examples/helloworld/-/issues/1) so try replacing 

```yaml
    container:
        "busybox:1.33"
```

```yaml
    container:
        "debian:bullseye-slim"
```

It will probably be easier to just make a new directory that is `recast_helloworld_reana` that just has the REANA workflow files in it than to have the `recast_helloworld` directory have both the original example and the REANA example in it like it is now.

Reply:
Command is: reana-client validate -f reana-snakemake.yaml
But the error message still the same:

==> ERROR: /Users/povsten/Documents/HelloWorld_RECAST/Snakemake-backend-for-RECAST-1/reana_helloworld/reana-snakemake.yaml is not a valid REANA specification:
Snakemake specification is invalid.
I was also trying to run the Root6Fit example from reana tutorial in same conda environment but the erro message is the same. But few weeks ago this example was working.

Reply:
We should also switch to "debian:bullseye-slim"